### PR TITLE
Feature/tan 17 symptom selectors

### PR DESCRIPTION
**Pull request by dan-kiss-dev-this**
symptom selector made

### CODE REVIEW

File: components/symptomTag.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import clsx from "clsx";
+import { useState } from "react";
+import { Switch } from "@headlessui/react";
+
+type SymptomTagProps = {
+  text: String;
+  selected?: boolean;
+  children?: any;
```
Comment (illepic) [inline components/symptomTag.tsx:8]: Type as `ReactNode`.

File: components/symptomTag.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import clsx from "clsx";
+import { useState } from "react";
+import { Switch } from "@headlessui/react";
+
+type SymptomTagProps = {
+  text: String;
```
Comment (illepic) [inline components/symptomTag.tsx:6]: Lower case, `String` isn't a type.

File: components/symptomTag.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  selected?: boolean;
+  children?: any;
+};
+
+const SymptomTag = ({
+  text,
+  selected = false,
+  children = null,
```
Comment (illepic) [inline components/symptomTag.tsx:14]: No default `null`.

File: components/symptomTag.tsx (hunk 1 above)
Comment (illepic) [reply components/symptomTag.tsx:8]: Not optional.

File: tailwind.config.js [hunk 4]
```diff
@@ -14,6 +14,9 @@ module.exports = {
           blurple: {
             500: "#5765F2",
           },
+          lightBlurple: {
+            500: "#C9CEFE",
+          },
```
Comment (illepic) [inline tailwind.config.js:19]: Move to `blerple.300` instead.

**Review by illepic — CHANGES_REQUESTED** [review]

File: components/selectorGroup.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  const [selectedIds, setSelectedIds] = useState<string[]>(initialSelected);
+
+  const toggleSelected = (id: string) => {
+    if (selectedIds.includes(id)) {
+      // already selected, so remove
+      setSelectedIds(selectedIds.filter((selectedId) => selectedId !== id));
+    } else {
+      // not yet selected, so add
+      setSelectedIds([...selectedIds, id]);
+    }
```
Comment (illepic) [inline components/selectorGroup.tsx:45]: Let's try and early return instead of else.

File: components/symptom.tsx [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  selected?: boolean;
+  children: ReactNode;
+};
+
+const Symptom = ({text, selected = false, children}: Props) => (
+  <div
+    className={clsx(
+      'inline-flex text-black font-bold py-4 px-5 rounded-t-lg rounded-b-lg border-2 border-tangie-blurple-500',
+      {
+        'bg-white-500  ': selected,
```
Comment (illepic) [inline components/symptom.tsx:15]: Remove double space

File: components/symptom.tsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        'bg-tangie-blurple-300': !selected,
+      },
+    )}
+  >
+    <div>{children} </div>
```
Comment (illepic) [inline components/symptom.tsx:20]: Extra space

**Review by illepic — CHANGES_REQUESTED** [review]

File: components/symptom.tsx (hunk 6 above)
Comment (paabloLC) [reply components/symptom.tsx:15]: this is done!

**Review by illepic — APPROVED** [review]

**Pull request merged**